adserver/decisionengine/backends.py

```diff
@@ -292,6 +293,10 @@ def filter_flight(self, flight):
         if flight.weighted_clicks_needed_this_interval() <= 0:
             return False
 
+        # Skip if the flight is not meant to show on these days
+        if not flight.show_to_day(timezone.now().strftime("%A").lower()):
```

Comment:
I guess locale will always be EN_US so this should *just work*.

Reply:
Ah, good point. You can also get it to return an `int`, and use that to map to the day, if we're worried about it.